app/views/categories/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2><%= @category.name %></h2>
+
+<strong>Categories:</strong>
+<% current_user.categories.each do |category| %>
+  <%= link_to category.name, category_path(category) %>
+<% end %>
+
+<%= render 'shared/todo_list' %>
```

Comment:
This partial is great! Love how this leverages the main todo list view

Reply:
Yes, I second that!

Reply:
I third that!